lib/config/presets/internal/group.ts

```diff
@@ -458,7 +458,6 @@ const staticGroups = {
       'group:react',
       'group:remark',
       'group:resilience4j',
-      'group:rubyOnRails',
```

Comment:
Do we want to setup "config migration" code for this change?

It might be nice if Renovate opens a config migration PR to:

1. Drop `group:rubyOnRails`
1. Add monorepo config for `monorepo:ruby-on-rails`

Reply:
Are there existing examples? If so I could give it a try.

Reply:
Let's wait for a maintainer to say if my idea is good/bad, before you start changing things. :wink:

Reply:
please leave this group in place, it does nothing by default